### PR TITLE
Add measurements and discrete measurements values and validity in equipments tab views

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java [hunk 1]
```diff
@@ -110,6 +112,21 @@ private static ThreeWindingsTransformerTabInfos toTabInfos(Identifiable<?> ident
...
+        builder.measurementP1(toMeasurement(threeWT, Measurement.Type.ACTIVE_POWER, 0))
+            .measurementQ1(toMeasurement(threeWT, Measurement.Type.REACTIVE_POWER, 0))
+            .measurementP2(toMeasurement(threeWT, Measurement.Type.ACTIVE_POWER, 1))
+            .measurementQ2(toMeasurement(threeWT, Measurement.Type.REACTIVE_POWER, 1))
+            .measurementP3(toMeasurement(threeWT, Measurement.Type.ACTIVE_POWER, 2))
+            .measurementQ3(toMeasurement(threeWT, Measurement.Type.REACTIVE_POWER, 2));
+
+        builder.measurementRatioTap1(toMeasurementTapChanger(threeWT, DiscreteMeasurement.Type.TAP_POSITION, DiscreteMeasurement.TapChanger.RATIO_TAP_CHANGER_1))
+            .measurementPhaseTap1(toMeasurementTapChanger(threeWT, DiscreteMeasurement.Type.TAP_POSITION, DiscreteMeasurement.TapChanger.PHASE_TAP_CHANGER_1))
+            .measurementRatioTap1(toMeasurementTapChanger(threeWT, DiscreteMeasurement.Type.TAP_POSITION, DiscreteMeasurement.TapChanger.RATIO_TAP_CHANGER_2))
```
Comment (achour94) [inline src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java:125]: shouldn't be measurementRatioTap2 ?

File: src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java [hunk 2]
```diff
@@ -110,6 +112,21 @@ private static ThreeWindingsTransformerTabInfos toTabInfos(Identifiable<?> ident
...
+            .measurementPhaseTap1(toMeasurementTapChanger(threeWT, DiscreteMeasurement.Type.TAP_POSITION, DiscreteMeasurement.TapChanger.PHASE_TAP_CHANGER_2))
```
Comment (achour94) [inline src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java:126]: measurementPhaseTap2

File: src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java [hunk 3]
```diff
@@ -110,6 +112,21 @@ private static ThreeWindingsTransformerTabInfos toTabInfos(Identifiable<?> ident
...
+            .measurementRatioTap1(toMeasurementTapChanger(threeWT, DiscreteMeasurement.Type.TAP_POSITION, DiscreteMeasurement.TapChanger.RATIO_TAP_CHANGER_3))
```
Comment (achour94) [inline src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java:127]: measurementRatioTap3

File: src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java [hunk 4]
```diff
@@ -110,6 +112,21 @@ private static ThreeWindingsTransformerTabInfos toTabInfos(Identifiable<?> ident
...
+            .measurementPhaseTap1(toMeasurementTapChanger(threeWT, DiscreteMeasurement.Type.TAP_POSITION, DiscreteMeasurement.TapChanger.PHASE_TAP_CHANGER_3));
```
Comment (achour94) [inline src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java:128]: measurementPhaseTap3

File: src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java]: :+1:  Fixed

File: src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java (hunk 3 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java]: :+1: Fixed

File: src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java (hunk 4 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/network/map/dto/mapper/ThreeWindingsTransformerInfosMapper.java]: :+1: Fixed

**Review by achour94 — APPROVED** [review]

**Pull request merged**